sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNode.scala

```diff
@@ -405,6 +405,80 @@ class WhileStatementExec(
   }
 }
 
+/**
+ * Executable node for RepeatStatement.
+ * @param condition Executable node for the condition.
```

Comment:
It would be nice if you provide more info about parameters like "... shall be evaluated to a row with single boolean value otherwise it throws an exception."